### PR TITLE
moinism's chrome cheat sheet

### PR DESCRIPTION
**Pull request by moinism**
**What does your Instant Answer do?**
It provides keyboard shortcuts about Google Chrome.

**What is the data source for your Instant Answer? (Provide a link if possible)**
https://support.google.com/chrome/answer/157179?hl=en

**Why did you choose this data source?**
Its the official Google help.

**Are there any other alternative (better) data sources?**
Nope.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Internet users looking to get more productive with their browsers.

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
No, couldn't find any.

**Are you having any problems? Do you need our help with anything?**
No.

**Where did you hear about DuckDuckHack? (For first time contributors)**
From DDG main site.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-08-03 at 22 28 44

-----
IA Page: https://duck.co/ia/view/chrome_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@moinism thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

File: share/goodie/cheat_sheets/chrome.json [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    ],
+
+    "sections": {
+        "Tab and Window": [
+            {
+                "key": "[Ctrl] [N]", 
+                "val": "Open new window"
+            },
+            {
+                "key": "[Ctrl-Shift] [N]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/chrome.json]: There's no need for the `-`. `[Ctrl] [Shift] [N]` means that all 3 are pressed together. If they're separated with a `,` then it means they're subsequent (not joint) keypresses, i.e. `[Ctrl] [F], [<search term>]`

**Comment by moinism** [comment]
So should I change, commit and create another pull request or is there any other way around?

**Comment by moinism** [comment]
Never mind, create a new diff.

**Comment by moollaza** [comment]
@moinism this LGTM :+1:

File: share/goodie/cheat_sheets/chrome.json [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+            },
+             {
+                "key": "[Ctrl] [H]",
+                "val": "Open history page"
+            },
+             {
+                "key": "[Ctrl] [J]",
+                "val": "Open downloads page"
+            },
+             {
```
Comment (moollaza) [inline share/goodie/cheat_sheets/chrome.json]: Just notice that your indentation is a little inconsistent. Please make sure the indents are all the same :smile:

**Pull request merged**